staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -114,7 +114,7 @@ type ObjectMeta struct {
 	// automatically. Name is primarily intended for creation idempotence and configuration
 	// definition.
 	// Cannot be updated.
-	// More info: http://kubernetes.io/docs/user-guide/identifiers#names
```

Comment:
it would probably be good to add redirects on the website as well to make the old URLs continue working - https://github.com/kubernetes/website/blob/main/static/_redirects

Reply:
I changed these URLs backup, and open a PR in kubernetes/website, shows below:
https://github.com/kubernetes/website/pull/39954